server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueTest.java

```diff
@@ -170,6 +170,32 @@ void browseShouldReturnCurrentlyEnqueuedMailFromAllSlices() throws Exception {
             "5-1", "5-2", "5-3", "5-4", "5-5");
     }
 
+    @Test
+    void dequeueShouldWorkAfterNetworkOutage() throws Exception {
+        String name = "myMail";
+        getMailQueue().enQueue(defaultMail()
+            .name(name)
+            .build());
+
+        rabbitMQExtension.getRabbitMQ().pause();
+
+        new Thread(() -> {
```

Comment:
Please always name threads so we can debug them once they leak.
Also, an executor seems more relevant than a thread here.